README.md

```diff
@@ -3,7 +3,6 @@
 [![Build Status](https://travis-ci.org/lean-delivery/tf-module-aws-core.svg?branch=master)](https://travis-ci.org/lean-delivery/tf-module-aws-core)
 
 ## Description
```

Comment:
MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines

Reply:
fixed at next commit